.changeset/poor-hats-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-uikit/design-system': patch
+'@commercetools-uikit/search-select-input': patch
+'@commercetools-uikit/select-utils': patch
+---
+
+Add new design tokens for search select input component.
```

Comment:
```suggestion
Add new design tokens for select input components.
```

The changeset should highlight the select-input components you worked on in this PR. Most of them are missing here.

Reply:
Thank you, 
Updated here: https://github.com/commercetools/ui-kit/pull/2373/commits/49476539925ae803f7cbad5b5d5ae9da7c71da16